### PR TITLE
py tracer fixes

### PR DESCRIPTION
**Pull request by agola11**
Not merging yet due to two issues:

* LLM/Chat Models inputs and outputs not showing up correctly rendered
* LLM/Chat Models extra showing up twice (one in rendered format, one in json format)

cc @vowelparrot

### CODE REVIEW

**Review by vowelparrot — APPROVED** [review]

**Review by vowelparrot — APPROVED** [review]

File: langchain/callbacks/tracers/langchain.py [hunk 1]
```diff
@@ -44,8 +75,24 @@ def _get_tenant_id(
         return tenant_id_
     endpoint_ = endpoint or get_endpoint()
     headers_ = headers or get_headers()
-    response = requests.get(endpoint_ + "/tenants", headers=headers_)
-    raise_for_status_with_text(response)
+    response = None
+    try:
+        response = requests.get(endpoint_ + "/tenants", headers=headers_)
```
Comment (nfcampos) [inline langchain/callbacks/tracers/langchain.py:80]: Shouldn't this be removed, or is that another PR?

File: langchain/callbacks/tracers/langchain.py [hunk 2]
```diff
@@ -72,6 +119,8 @@ def __init__(
         self.example_id = example_id
         self.session_name = session_name or os.getenv("LANGCHAIN_SESSION", "default")
         self.session_extra = session_extra
+        # set max_workers to 1 to process tasks in order
```
Comment (nfcampos) [inline langchain/callbacks/tracers/langchain.py:122]: Why does order matter? Can we rely on every user of the api being as strict about order as we are being here?

Comment (vowelparrot) [reply langchain/callbacks/tracers/langchain.py:122]: It'll fail if a child run gets posted before a parent run at least, right? (parent_run_id not found)

File: langchain/callbacks/tracers/langchain.py (hunk 1 above)
Comment (vowelparrot) [reply langchain/callbacks/tracers/langchain.py:80]: https://github.com/hwchase17/langchain/pull/5198

**Pull request merged**